cli/render.go

```diff
@@ -127,14 +135,20 @@ func maybeCreateDestinationDir(path string) error {
 	return nil
 }
 
-func writeFile(path string, content string, overwrite bool) error {
+func writeFile(c *RenderCommand, path string, content string) error {
 	// Check to see if the file already exists and validate against the value
 	// of overwrite.
+	var overwrite bool
```

Comment:
We don't use this value outside the `if err == nil {` scope right? So we can probably just walrus (`:=`) it there.